web/packages/teleport/src/Discover/Database/EnrollRdsDatabase/AutoEnrollment.tsx

```diff
@@ -59,7 +59,7 @@ const emptyTableData = (): TableData => ({
   fetchStatus: 'disabled',
   instancesStartKey: '',
   clustersStartKey: '',
-  oneOfError: '', // todo handle
+  oneOfError: '',
 });
 
 export function AutoEnrollment({
```

Comment:
I think it'd still be beneficial to add `key` as a required prop to both `AutoEnrollment` and `SingleEnrollment`, similar to how I suggested this in the patch (https://github.com/gravitational/teleport/pull/44671#discussion_r1696936847). The component is not going to behave properly without it and it's easy to omit the key during refactoring.